### PR TITLE
feat(protocol-designer): remove secondary navbar

### PR DESCRIPTION
**Pull request by koji**
# Overview
remove secondary navbar

close AUTH-662

## Risk assessment
low

### CODE REVIEW

File: protocol-designer/src/__tests__/NavigationBar.test.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    render()
+    screen.getByText('Opentrons')
+    screen.getByText('Protocol Designer')
+    screen.getByText('Version # fake_PD_version')
+    screen.getByText('Create new protocol')
+    screen.getByText('Import')
+  })
+
+  it.todo(
+    'when clicking Create new protocol, mock function should be called',
```
Comment (jerader) [inline protocol-designer/src/__tests__/NavigationBar.test.tsx:32]: is it possible to write these todos now? you shouldn't be blocked from them right?

**Review by jerader — APPROVED** [review]
lgtm, just left 1 question.

Also, fyi, now there is no way to access the liquids page but that is fine for now. Current designs don't need a liquids page but i am convinced with the liquid class support, which will be after the redesign, we will need the liquids page. Planning to discuss more with Skye & Felix about it to get a solid answer.

File: protocol-designer/src/__tests__/NavigationBar.test.tsx (hunk 1 above)
Comment (koji) [reply protocol-designer/src/__tests__/NavigationBar.test.tsx:32]: tried to test them but they were a little bit tricky to test and i made them todo temporarily.

Comment (jerader) [reply protocol-designer/src/__tests__/NavigationBar.test.tsx:32]: cool that is fine for now! feel free to merge after you resolve the merge conflict

**Comment by koji** [comment]
- [ ] fix test error

**Pull request merged**